packages/icons/README.md

```diff
@@ -0,0 +1,3 @@
+# Design System Icons
+
+TODO
```

Comment:
Are there any tickets open for the TODOs?

Reply:
I added a readme and a contributing.md